File: tuikit/src/main/java/com/tencent/qcloud/tim/uikit2/modules/chat/base/BaseInputFragment.java
package com.tencent.qcloud.tim.uikit2.modules.chat.base;

import com.tencent.qcloud.tim.uikit2.base.BaseFragment;
import com.tencent.qcloud.tim.uikit2.modules.chat.interfaces.IChatLayout;

public class BaseInputFragment extends BaseFragment {

    private IChatLayout mChatLayout;

    public IChatLayout getChatLayout() {
        return mChatLayout;
    }

    public BaseInputFragment setChatLayout(IChatLayout layout) {
        mChatLayout = layout;
        return this;
    }
}

File: im/src/main/java/com/black/im/fragment/BaseInputFragment.kt
package com.black.im.fragment

import com.black.im.interfaces.IChatLayout

open class BaseInputFragment : BaseFragment() {
    var chatLayout: IChatLayout? = null
        private set

    fun setChatLayout(layout: IChatLayout?): BaseInputFragment {
        chatLayout = layout
        return this
    }
}
